File: Java/Display.java
public class Display extends PartLaptop {
    private String resolusi;
    private int ukuran;
    private String tipePanel;
    private String aspekRatio;

    public Display() {
        super();
        this.resolusi = "";
        this.ukuran = 0;
        this.tipePanel = "";
        this.aspekRatio = "";
    }

    public Display(int id, String nama, int harga, int stok,
                   String noPart, String manufaktur, String kondisi,
                   String resolusi, int ukuran,
                   String tipePanel, String aspekRatio) {
        super(id, nama, harga, stok, noPart, manufaktur, kondisi);
        this.resolusi = resolusi;
        this.ukuran = ukuran;
        this.tipePanel = tipePanel;
        this.aspekRatio = aspekRatio;
    }

    public String getResolusi() { return resolusi; }
    public void setResolusi(String resolusi) { this.resolusi = resolusi; }

    public int getUkuran() { return ukuran; }
    public void setUkuran(int ukuran) { this.ukuran = ukuran; }

    public String getTipePanel() { return tipePanel; }
    public void setTipePanel(String tipePanel) { this.tipePanel = tipePanel; }

    public String getAspekRatio() { return aspekRatio; }
    public void setAspekRatio(String aspekRatio) { this.aspekRatio = aspekRatio; }
}

File: Python/Display.py
from PartLaptop import PartLaptop

class Display(PartLaptop):
    def __init__(self,
                 id=0, nama="", harga=0, stok=0,
                 no_part="", manufaktur="", kondisi="",
                 resolusi="", ukuran=0,
                 tipe_panel="", aspek_ratio=""):
        super().__init__(id, nama, harga, stok, no_part, manufaktur, kondisi)
        self._resolusi = resolusi
        self._ukuran = ukuran
        self._tipe_panel = tipe_panel
        self._aspek_ratio = aspek_ratio

    @property
    def resolusi(self): return self._resolusi
    @resolusi.setter
    def resolusi(self, v): self._resolusi = v

    @property
    def ukuran(self): return self._ukuran
    @ukuran.setter
    def ukuran(self, v): self._ukuran = v

    @property
    def tipe_panel(self): return self._tipe_panel
    @tipe_panel.setter
    def tipe_panel(self, v): self._tipe_panel = v

    @property
    def aspek_ratio(self): return self._aspek_ratio
    @aspek_ratio.setter
    def aspek_ratio(self, v): self._aspek_ratio = v
